src/Spago/Purs.hs

```diff
@@ -54,13 +54,36 @@ bundle withMain (ModuleName moduleName) (TargetPath targetPath) = do
     ("Bundle failed.")
 
 
-docs :: Spago m => [SourcePath] -> m ()
-docs sourcePaths = do
+data DocsFormat
+  = Html
+  | Markdown
+  | Ctags
+  | Etags
+
+parseDocsFormat :: Text -> Maybe DocsFormat
+parseDocsFormat = \case
+  "html"     -> Just Html
+  "markdown" -> Just Markdown
+  "ctags"    -> Just Ctags
+  "etags"    -> Just Etags
+  _          -> Nothing
+
+printDocsFormat :: DocsFormat -> Text
+printDocsFormat = \case
+  Html     -> "html"
+  Markdown -> "markdown"
+  Ctags    -> "ctags"
+  Etags    -> "etags"
+
+
+docs :: Spago m => Maybe DocsFormat -> [SourcePath] -> m ()
+docs format sourcePaths = do
   let
     paths = Text.intercalate " " $ Messages.surroundQuote <$> map unSourcePath sourcePaths
-    cmd = "purs docs " <> paths <> " --format html"
+    formatStr = printDocsFormat $ fromMaybe Html format
+    cmd = "purs docs " <> paths <> " --format " <> formatStr
   runWithOutput cmd
-    ("Docs generated. Index is at " <> Messages.surroundQuote "./generated-docs/index.html")
+    "Docs generation succeeded."
```

Comment:
I'd like to keep informing the user on where the files are generated, so maybe here we could print different things depending on the `DocsFormat` value?

Reply:
Ah, in 0.13 the compiler prints the location of the files for you. For example:

- `Documentation written to: generated-docs/html`
- `Documentation written to: tags`

Then we get our output underneath.

Reply:
Oh that's great!